proof/refine/ARM/StateRelation.thy

```diff
@@ -192,13 +196,20 @@ where
   \<and> tcb_bound_notification tcb = tcbBoundNotification tcb'
   \<and> tcb_mcpriority tcb = tcbMCP tcb'"
 
+\<comment> \<open>
+  A pair of objects @{term "(obj, obj')"} should satisfy the follow relation when, under further
+  mild assumptions, a @{term corres_underlying} lemma for @{term "set_object obj"}
+  and @{term "setObject obj'"} can be stated: see setObject_other_corres below.
```

Comment:
Should this be "see setObject_other_corres in KHeap_R"?

Reply:
Yup! Will change